package.json

```diff
@@ -72,6 +72,7 @@
     "react-ga": "^2.2.0",
     "react-hot-loader": "next",
     "react-markdown": "^2.5.0",
+    "react-mdl": "^1.11.0",
```

Comment:
we depend currently on a library called [spinkit](http://tobiasahlin.com/spinkit/) for the 'loading' thingy we already use - do you think there's a spinner there we could use so we can avoid depending on `react-mdl`?

Reply:
I'm fine with it. @roberthouse54 do you see anything on that page which fits the design or do you prefer the MDL spinner?

Reply:
Lets use this one:
![image](https://user-images.githubusercontent.com/20047260/33498552-c5f6a994-d69f-11e7-985a-933b4fbf0d18.png)